src/Control/Monad/Trans/Except.hs

```diff
@@ -0,0 +1,108 @@
+{-# LANGUAGE PackageImports #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{- | A newtype around @transformers@ 'Transformers.ExceptT' with less restrictive 'Applicative', 'Selective', and 'Alternative' implementations.
+
+This tries to copy verbatim the API from @transformers@,
+so it can be used as a drop-in replacement.
+The documentation can be found in the [@transformers@](https://hackage.haskell.org/package/transformers/docs/Control-Monad-Trans-Except.html) package.
+-}
+module Control.Monad.Trans.Except where
```

Comment:
I'm not sure whether that will be a problematic name clash if a project has dependencies on both `transformers` and `selective`. Package qualified imports help in such a situation, but I don't know whether we would want that. The alternative is to rename this module.

Reply:
I think I prefer to rename this module to avoid such conflicts. Let's keep the library's module inside the `Control.Selective` umbrella.

Reply:
Perhaps, `Control.Selective.Transformers`?

Reply:
I'd prefer `Control.Selective.Trans.Except`. That would be directly analogous.

Reply:
Because it actually is a `Selective` transformer :smile:

Reply:
Yes, I like `Control.Selective.Trans.Except`!